### ISSUE TITLE
New transactions don't show up

### ISSUE THREAD

**Issue by tschubotz** [gnosis/safe-client-gateway#307]
**Describe the bug**

**To Reproduce**
- Post new tx to tx service -> https://safe-transaction.rinkeby.gnosis.io/api/v1/safes/0xaE3c91c89153DEaC332Ab7BBd167164978638c30/multisig-transactions/ has some not executed txs.

They don't show up at https://safe-client.rinkeby.gnosis.io/v1/safes/0xaE3c91c89153DEaC332Ab7BBd167164978638c30/transactions/queued/ 

**Expected behavior**
- https://safe-client.rinkeby.gnosis.io/v1/safes/0xaE3c91c89153DEaC332Ab7BBd167164978638c30/transactions/queued/ returns the txs waiting for confirmations / execution

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Firefox

**Comment by tschubotz** [comment]
@rmeissner @jpalvarezl is this related to the caching issues? If yes, could you please invalidate the cache for my Safe?

**Comment by jpalvarezl** [comment]
Seems to be. I invalidated the cache and now some transactions are returned. This is a bit weird, as yesterday we rolled back the gateway to `v1.8.3` but it seems it was only `mainnet`. I checked the about endpoint for `rinkeby` and `xdai` and the version is still `v1.9.0`. Today we plan on releasing a hotfix to all prod environments, so all production networks should have the same deployed version again.
